host_classifer.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
you misspelled your file name, it should be 'host_classifier.sh'

also, your output format doesn't match what's defined in the document (need correct spacing and equals sign)

Reply:
The file name is definitely corrected.

The output spacing now has `=` signs that are aligned and i believe spacing matches what's in the training confluence